### PR TITLE
Test gaze estimation

### PR DESCRIPTION
**Pull request by gluhar2006**

### CODE REVIEW

File: tests/test_estimate_gaze.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def setup_class(cls):
+        super().setup_class()
+        cls.defaultDetector = cls.faceEngine.createFaceDetector(DetectorType.FACE_DET_DEFAULT)
+        cls.warper = cls.faceEngine.createWarper()
+        cls.gazeEstimator = cls.faceEngine.createGazeEstimator()
+        cls.faceDetection = cls.defaultDetector.detectOne(VLImage.load(filename=ONE_FACE))
+        cls.warp = cls.warper.warp(faceDetection=cls.faceDetection)
+
+    @staticmethod
+    def validate_emotion_dict(receivedDict: dict):
```
Comment (matemax) [inline tests/test_estimate_gaze.py]: validate_emotion_dict

File: tests/test_estimate_gaze.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        """
+        Validate emotion dict
+        """
+        assert sorted(["pitch", "yaw"]) == sorted(receivedDict.keys())
+        for gaze in ("pitch", "yaw"):
+            assert -180 <= receivedDict[gaze] <= 180
```
Comment (matemax) [inline tests/test_estimate_gaze.py:33]: check type of validate_emotion_dict

File: tests/test_estimate_gaze.py (hunk 1 above)
Comment (gluhar2006) [reply tests/test_estimate_gaze.py]: fixed

File: tests/test_estimate_gaze.py (hunk 2 above)
Comment (gluhar2006) [reply tests/test_estimate_gaze.py:33]: ok

File: tests/test_estimate_gaze.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    def test_estimate_gaze_landmarks5(self):
+        """
+        Test gaze estimator with landmarks 5
+        """
+        for detector in self.detectors:
+            with self.subTest(detectorType=detector.detectorType):
+                landMarks5Transformation = self.warper.makeWarpTransformationWithLandmarks(self.faceDetection, "L5")
+                gazeEstimation = self.gazeEstimator.estimate(landMarks5Transformation, self.warp).asDict()
```
Comment (matemax) [inline tests/test_estimate_gaze.py:42]: + test without transformation

Comment (gluhar2006) [reply tests/test_estimate_gaze.py:42]: added

**Pull request merged**